requirements.txt

```diff
@@ -1,4 +1,6 @@
 flake8
 py-solc-x
 pytest
-oasis-sapphire-py
\ No newline at end of file
+pytest-asyncio
+oasis-sapphire-py
```

Comment:
Let's pin this.
```suggestion
oasis-sapphire-py=0.4.*
```

Reply:
Removed leftover use_async option from the README, pinned oasis-sapphire-py.